src/seedu/addressbook/ui/Formatter.java

```diff
@@ -63,4 +64,15 @@ private static String getIndexedListItem(int visibleIndex, String listItem) {
         return String.format(MESSAGE_INDEXED_LIST_ITEM, visibleIndex, listItem);
     }
 
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    public static String getPrintableString(Printable... printables){
```

Comment:
Good that you remembered to include header comments for this new method.

Perhaps also consider where `getPrintableString` method could be used in your code!